packages/wallet-sdk/src/components/RedirectDialog/RedirectDialog.tsx

```diff
@@ -0,0 +1,67 @@
+import clsx from "clsx";
+import { FunctionComponent, h, render } from "preact";
+
+import { injectCssReset } from "../../lib/cssReset";
+import { SnackbarContainer } from "../Snackbar";
+import css from "./RedirectDialog-css";
+
+type RedirectDialogProps = {
+  title: string;
+  buttonText: string;
+  onButtonClick: () => void;
+};
+
+export class RedirectDialog {
```

Comment:
to show a button in case of redirection getting blocked by browser
![image](https://github.com/coinbase/coinbase-wallet-sdk/assets/966662/73deace4-1f84-4149-854a-3da02d781da0)